### PR TITLE
Remove autocreate plugin

### PR DESCRIPTION
**Pull request by jeffpc**

### CODE REVIEW

File: source/settings/plugin/autocreate-plugin.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@
 autocreate plugin
 ===================
 
+.. versionremoved:: 2.3.14 Use autocreate namespace mailbox setting instead.
+
 With v2.1+ you don't need this plugin. Use `mailbox { auto } <https://wiki.dovecot.org/MailboxSettings>`_ setting instead.
```
Comment (sirainen) [inline source/settings/plugin/autocreate-plugin.rst:9]: These two lines could be combined in some way. Also would be really nice if the wiki text was moved to doc :)

Comment (sirainen) [reply source/settings/plugin/autocreate-plugin.rst:9]: They could both link to https://doc.dovecot.org/configuration_manual/namespace/ - and even better if there was #auto anchor.

Comment (sirainen) [reply source/settings/plugin/autocreate-plugin.rst:9]: or maybe https://doc.dovecot.org/configuration_manual/namespace/#mailbox-settings

Comment (cmouse) [reply source/settings/plugin/autocreate-plugin.rst:9]: @jeffpc  ping

Comment (jeffpc) [reply source/settings/plugin/autocreate-plugin.rst]: Like that?

**Review by sirainen — DISMISSED** [review]

**Review by sirainen — APPROVED** [review]

**Pull request merged**